scripts/qa_crowbarsetup.sh

```diff
@@ -3816,6 +3880,13 @@ function oncontroller_testsetup()
         tempestret=$?
     fi
 
+    # Run Horizon integration test if configure to do so
+    integrationret=0
+    if [ "$want_integration_test" = "1" ]; then
```

Comment:
maybe we want to label that want_horizon_integration_test ? or do we plan to reuse that? then maybe it should be ..._tests  instead?